verify/src/main/java/com/twilio/verify/TwilioVerify.kt

```diff
@@ -79,7 +79,7 @@ interface TwilioVerify {
   )
 
   /**
-   * Gets all **Factors** created by the app
+   * Gets all **Factors** created by the app, this method will return the factors in local data.
```

Comment:
I think this should be `local storage` instead of `local data`

Reply:
Done, thanks